README.md

```diff
@@ -33,7 +33,7 @@ We envision a product with user-customizable views of jobs running, insights int
 
 The Job Manager [defines an API](api/jobs.yaml) via OpenAPI. An Angular2 UI is provided over the autogenerated Typescript bindings for this API. The UI is configurable at compilation time to support various deployment environments (see [environment.ts](ui/src/environments/environment.ts)), including auth, cloud projects, and label columns.
 
-The UI must be deployed along with a backend implementation of the API; two such implementations are provided here:
+The UI must be deployed along with a backend implementation of the API and two such implementations are provided here:
```

Comment:
nit: Would prefer just a "," instead of "and"

Reply:
I will close the PR and do it again.